protocol/get_events.go

```diff
@@ -25,6 +25,7 @@ type EventInfo struct {
 	LedgerClosedAt string `json:"ledgerClosedAt"`
 	ContractID     string `json:"contractId"`
 	ID             string `json:"id"`
+	OpIndex        uint32 `json:"operationIndex"`
```

Comment:
I think we should also include the transaction index . With the ledger sequence, transaction index, and operation index I think we have all the information needed to sort events

Reply:
Might as well! Done in 49fd1e8